### DISCUSSION TITLE
MAS not updating, non steam, HELP

### DISCUSSION THREAD

**Discussion by EColi07** [Monika-After-Story/MonikaModDev#9605] [QA - tech support + general] [answered]
I was originally using steam but then I downloaded DDLC again for backup, this time without using steam. But when I click on settings and then update version, the game says that there's a new update available and after I click "update" it tells me that "Monika After Story is now up to date", and it gives me two options: Quit and Restart.

I've tried both but every time I open the game it keeps saying that a new update is available and the game seems to have never updated itself.

Thanks for reading :)

**Comment by ThePotatoGuy** [comment, marked as answer]
follow the steps on [this wiki](https://github.com/Monika-After-Story/MonikaModDev/wiki/FAQ#mas-keeps-telling-me-that-theres-an-update-but-when-i-click-update-now-it-says-mas-is-up-to-date)
